js/app.js

```diff
@@ -114,71 +114,76 @@ cardItem.forEach((item, j) => {
   callCardBody(j);
```

Comment:
- [ ] In the ```cardItem``` array each object should contain at least the following information about the project: 
  ```name```,
  ```description```,
  ```featured image```,
  ```technologies```,
  ```link to live version```,
  ```link to the project source code```,
 please consider adding the ```live link``` and ```source link``` to the objects inside the array so you meet the requirement